lib/holiday_status.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module DiscourseCalendar
+  class HolidayStatus
+    EMOJI ||= 'desert_island'
+
+    def self.is_holiday_status?(status)
+      status.emoji == EMOJI && status.description == I18n.t("discourse_calendar.holiday_status.description")
```

Comment:
Good point, it's definitely fragile, and there will be a strange behavior in case the copy is changed. But I think it's good enough for the first version. Most likely, we'll end up with adding a `set_by` column to the user_statuse table.

> Is the description always set using forum's default lang?

Yes, it uses the default locale of the site (I double-checked it).